### ISSUE TITLE
Accessibility: Reader Settings

### ISSUE THREAD

**Issue by jd-alexander** [wordpress-mobile/WordPress-Android#10731]
###### Accessibility Scanner Audit

<img src="https://user-images.githubusercontent.com/1509205/68079747-d5389180-fdab-11e9-9c2e-879189447169.png" width="400" alt="settings" />

* Followed Sites Tab - Text contrast

	The item's text contrast ratio is 2.04. This ratio is based on an estimated foreground color of `#3895BA` and an estimated background color of `#006088`. Consider increasing this item's text contrast ratio to 3.00 or greater.
	
* Sites You May Like Tab - Text contrast

	The item's text contrast ratio is 2.04. This ratio is based on an estimated foreground color of `#3895BA` and an estimated background color of `#006088`. Consider increasing this item's text contrast ratio to 3.00 or greater.
		

* Remove Button `org.wordpress.android:id/btn_remove` - Touch target
	
	This item's size is 32dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.
	
* Add URL or Tag EditText org.wordpress.android:id/edit_add - Touch target

	This item's height is 38dp. Consider making the height of this touch target 48dp or larger.
	
* Add Button org.wordpress.android:id/btn_add - Touch target

	This item's size is 32dp x 32dp. Consider making this touch target 48dp wide and 48dp high or larger.
	
* Add URL or Tag EditText `org.wordpress.android:id/edit_add` - Text contrast

	The item's text contrast ratio is 4.20. This ratio is based on an estimated foreground color of `#787C82` and an estimated background color of `#FFFFFF`. Consider using a contrast ratio greater than 4.50 for small text, or 3.00 for large text.
		
###### Talkback Audit 
    	
* Search  
<img src="https://user-images.githubusercontent.com/1509205/68091812-5b4ee980-fe39-11e9-8bca-e893b2ff0d21.png" alt="Screenshot_20191102-200643" width="400" />

* Search input doesn't tell when no results have been returned. Return results after waiting for at least 4 - 5 seconds to ensure query was properly entered. 	
	
* When search returns results the user won't know if results are actually present. The app should announce that two items actually exist. (It actually announces this the first time the user selects an item in the list.)

### Sites You May Like Tab

* No issues noticed.
